.npmignore

```diff
@@ -1,6 +1,12 @@
-node_modules/
-src/
-build/
+node_modules
+src
+build
 .arcconfig
-.babelrc
 *.yml
+.editorconfig
+karma.config.js
+travis.yml
+CHANGELOG.md
+CONTRIBUTING.md
+lib/test
```

Comment:
ah, I see, so this will limit people to have to clone the repo to contribute.  I'm okay with that.  Now, only one way to do it, it's consistent.  Thanks.